src/icons/BrandIcon.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+
+import {useTheme} from '../context/theme';
+import {Theme} from '../models';
+import {a11yHiddenSvgProps} from '../utils/svg';
+
+export const BrandIcon: React.FC<React.SVGProps<SVGSVGElement>> = (props) => {
+    const theme = useTheme();
```

Comment:
it seems to me that we need to create two icons and decide which one to use in this [place](https://github.com/gravity-ui/page-constructor/pull/1084/files#diff-5af56b1085dd18b3dc134f9c9bd4e4a927a53823224377619050802b802d0828R21). I think it will be more obvious why it is changing.